c8_calc.rb

```diff
@@ -0,0 +1,50 @@
+puts "Let's crunch some numbers!"
+puts "---------------------------------"
+
+def operation
+  operator_hash = {"add" => '+', "+" => "+", "subtract" => "-" , "-" => "-", "multiply" => "*", "*" => "*", "divide" => "/", "/" => "/"}
+  print "Enter an operator: "
+  math_verb = gets.chomp.downcase
+  until operator_hash.has_key?(math_verb)
+    print "Enter an operator: "
+    math_verb = gets.chomp.downcase
+  end
+  return operator_hash[math_verb]
+end
+
+def num_input
+  print "Enter a number: "
+  number = Float(gets.chomp) rescue nil
```

Comment:
Do be careful when doing this.  We haven't talked about rescue yet, but later on a solution like this for other things could be frowned upon.  More info is [here](http://www.nickdesteffen.com/blog/why-rescue-nil-is-bad)